### ISSUE TITLE
dotnet-sdk on OSX

### ISSUE THREAD

**Issue by acesnik** [conda-forge/dotnet-feedstock#15]
Issue:
It looks like dotnet is intended to work on macOS, including the hashes being listed in `meta.yaml`: https://github.com/conda-forge/dotnet-feedstock/blob/master/recipe/meta.yaml#L3. However, when I try to install version 3.1.403, which installs properly on various linux machines, I get the following on macOS Big Sur:
```
Collecting package metadata (repodata.json): done
Solving environment: failed

ResolvePackageNotFound: 
  - dotnet-sdk=3.1.403
```

It also looks like the badge for OSX/macOS is missing from the installer page (https://anaconda.org/conda-forge/dotnet-sdk):
![image](https://user-images.githubusercontent.com/16342951/106825364-04ac0d00-6685-11eb-8dd2-ee92baa9ac2c.png)

Thanks for working on the dotnet conda installers! Let me know if you have any thoughts on how to get this installed on macOS.

-Tony

Environment (<code>conda list</code>): 
[conda_list.txt](https://github.com/conda-forge/dotnet-feedstock/files/5922221/conda_list.txt)

Details about  <code>conda</code> and system ( <code>conda info</code> ):
[conda_info.txt](https://github.com/conda-forge/dotnet-feedstock/files/5922226/conda_info.txt)

**Comment by dhirschfeld** [comment]
Just commenting to follow up shortly...

**Comment by johnwason** [comment]
I am also interested in using this package on OSX.

**Comment by acesnik** [comment]
https://github.com/conda-forge/dotnet-feedstock/pull/17

**Comment by dhirschfeld** [comment]
Sorry for the delay - will get to it this weekend...

**Comment by acesnik** [comment]
No worries!

**Comment by dhirschfeld** [comment]
Fixed by #17
